varya/functions.php

```diff
@@ -308,6 +308,6 @@ function varya_editor_theme_variables() {
 /**
  * Load default styles for Varya parent-theme-only
  */
-if ( ! is_child_theme() ) {
+//if ( ! is_child_theme() ) {
 	require get_template_directory() . '/default-style/functions.php';
```

Comment:
This will be a separate PR, but did we decide to move `default-styles` into Varya itself?

Asking again, because this wouldn't needed if we either a) integrated the `default styles` directly into Varya or b) moved them into this child theme.

Reply:
Yes, I believe we did decide to move the default styles into Varya itself. The "vanilla" styles can either live as an alternate set of files/rules in there for now too (essentially the opposite of the current approach), or we can spin them out into a separate child theme. Let's open another issue to track that.

Reply:
https://github.com/Automattic/themes-workspace/issues/20